### DISCUSSION TITLE
Automated source code docu like PyDoctor or Doxygen?

### DISCUSSION THREAD

**Discussion by buhtz** [mkdocs/mkdocs#3585] [Q&A] [answered]
Is it possible to generate automated source code docu with MkDoc like you can do it with PyDoctor or Doxygen?

**Comment by pawamoy** [comment]
Check out the mkdocstrings plugin :) https://github.com/mkdocstrings/mkdocstrings

**Reply by buhtz** [reply]
Thanks for your reply.
Is this a mkdocs-maintainer answer?

**Reply by pawamoy** [reply, marked as answer]
I'm part of the MkDocs team (I do almost only issue triaging), but I'm also the author of mkdocstrings. For more objectivity, check out the catalog: https://github.com/mkdocs/catalog/?tab=readme-ov-file#-api-documentation-building :smile:

**Reply by buhtz** [reply]
Thanks for your helpful reply. OK, now I am in the picture.
